modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -29,10 +29,10 @@
 //
 module LocaleModel {
 
-  use LocaleModelHelpFlat;
-  use LocaleModelHelpMem;
+  public use LocaleModelHelpFlat;
+  public use LocaleModelHelpMem;
```

Comment:
I think these are OK because they only contain `chpl_`, symbols right?

Reply:
They probably aren't problematic, but there seems to be something of a rats nest around the locale model-related modules that I wanted to detangle better, which is why I didn't flag them with a comment as being reviewed/necessary.  I wanted to better understand the dependencies between locale model-related modules and see whether we could clean them up a bit.

Reply:
Also noted in https://github.com/Cray/chapel-private/issues/1184 though they aren't necessarily problematic.